crates/oxc_span/src/atom.rs

```diff
@@ -12,6 +12,10 @@ pub struct Atom(CompactString);
 const BASE54_CHARS: &[u8; 64] = b"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ$_0123456789";
 
 impl Atom {
+    pub const fn new_inline(s: &'static str) -> Self {
+        Self(CompactString::new_inline(s))
+    }
```

Comment:
This is not used yet.

Reply:
Sorry, this got committed from my working branch